packages/mdc-chips/_variables.scss

```diff
@@ -19,3 +19,9 @@ $mdc-chip-fill-color-default: rgba(black, .08);
 $mdc-chip-ink-color-default: rgba(black, .87);
 $mdc-chip-horizontal-padding: 12px;
 $mdc-chip-vertical-padding: 7px;
+$mdc-chip-with-icon-padding: 4px;
+
+$mdc-chip-icon-color: rgba(0, 0, 0, .54);
```

Comment:
I normally prefer to use the global theme variables when possible (e.g., `text-primary-on-background`), because it makes it much easier for clients to create a dark theme.

In this case, though, some of the other values in this file don't map directly to theme vars (e.g., `.08`, `.62`), so it might not be feasible.

Just something to keep in mind in the future 🙂

Reply:
Replaced `mdc-chip-ink-color-default` with `text-primary-on-light`! I kept the icon colors though since the global variables are supposed to be for text.